ui/js/component/form.js

```diff
@@ -106,26 +113,45 @@ export class FormField extends React.PureComponent {
     delete otherProps.className;
     delete otherProps.postfix;
     delete otherProps.prefix;
-
-    const element = (
-      <this._element
-        id={elementId}
-        type={this._type}
-        name={this.props.name}
-        ref="field"
-        placeholder={this.props.placeholder}
-        className={
-          "form-field__input form-field__input-" +
-          this.props.type +
-          " " +
-          (this.props.className || "") +
-          (isError ? "form-field__input--error" : "")
-        }
-        {...otherProps}
-      >
-        {this.props.children}
-      </this._element>
-    );
+    const element = this._element === "SimpleMDE"
+      ? <SimpleMDE
+          id={elementId}
+          type={this._type}
+          name={this.props.name}
+          ref="field"
+          placeholder={this.props.placeholder}
+          className={
+            "form-field__input form-field__input-" +
+            this.props.type +
+            " " +
+            (this.props.className || "") +
+            (isError ? "form-field__input--error" : "")
+          }
+          name={this.props.name}
+          options={{
+            hideIcons: ["heading"],
+          }}
+          {...otherProps}
+        >
+          {this.props.children}
+        </SimpleMDE>
+      : <this._element // can't be applied to SimpleMDE idk why
```

Comment:
I'm decently confident this is possible, maybe with `this._element = <SimpleMDE>` or `this._element = SimpleMDE`? Worth looking into. If it's not, this can still be cleaned up by compacting the shared properties to one declaration and passing them in with an extract operator.